src/python/pants/engine/platform.py

```diff
@@ -20,9 +21,9 @@ def current(cls) -> 'Platform':
 
   @memoized_property
   def runtime_lib_path_env_var(self) -> str:
-    return self.match({
-      Platform.darwin: "DYLD_LIBRARY_PATH",
-      Platform.linux: "LD_LIBRARY_PATH",
+    return enums.match(self, {
+      self.darwin: "DYLD_LIBRARY_PATH",
+      self.linux: "LD_LIBRARY_PATH",
```

Comment:
Nit: I'd personally prefer these to still have the full name `Platform.x` as a style preference, because they are static and to me `self` means "per instance".

Reply:
Thank you -- this is great! It also makes it easier to search for instances of matching against `Platform` enums in the codebase!